### ISSUE TITLE
[blog: multiarch] "Push" does not succeed in the example procedure

### ISSUE THREAD

**Issue by tnk4on** [containers/podman.io_old#469]
I ran the following sample according to the contents of [the blog](https://podman.io/blogs/2021/10/11/multiarch.html). The push does not succeed with this procedure. (My repository is public and empty)
```
$ platarch=linux/amd64,linux/arm64
$ buildah build --jobs=2 --platform=$platarch --manifest test .
$ podman tag localhost/test:latest quay.io/tnk4on/test:latest
$ podman manifest rm localhost/test
$ podman manifest push --all quay.io/tnk4on/test docker://quay.io/tnk4on/test
Error: error retrieving local image from image name quay.io/tnk4on/test:latest: quay.io/tnk4on/test:latest: image is not a manifest list
```

This will work.
```
$ platarch=linux/amd64,linux/arm64
$ buildah build --jobs=2 --platform=$platarch --manifest test .
$ podman manifest push --all localhost/test docker://quay.io/tnk4on/test
```

Doesn't `podman tag` support manifests?
```
$ podman images
REPOSITORY                                   TAG             IMAGE ID      CREATED             SIZE
localhost/test                               latest          afd30818a571  About a minute ago  1.08 kB
<none>                                       <none>          f22909322136  About a minute ago  269 MB
quay.io/tnk4on/test                          latest          77e47b467966  About a minute ago  233 MB

$ podman version
Version:      3.4.0
API Version:  3.4.0
Go Version:   go1.16.8
Built:        Fri Oct  1 04:40:21 2021
OS/Arch:      linux/amd64
```

**Comment by TomSweeneyRedHat** [comment]
@tnk4on TYVM for the issue.  @cevich Can you tweak your blog please?

**Comment by cevich** [comment]
> Doesn't podman tag support manifests?

Hmmm, there was an issue in buildah about this, it is possible it didn't get fixed in `3.4`.  Def. try using `podman manifest tag`.

I'll investigate and adjust the article accordingly.  Thanks for reporting.

**Comment by cevich** [comment]
I was able to reproduce this problem.  So it seems `buildah tag` works fine on manifest lists but that fix never made it into podman `3.4`.  *Sigh*...the woes of writing a blog while releases are going out the door.  I'll update the article accordingly.  Thanks again for pointing out the problem.

**Comment by cevich** [comment]
@tnk4on FYI - A fix for this in podman is going to be backported into an upcoming `v3.4.x` release (date unknown).
